src/components/TrailsList/TrailsList.js

```diff
@@ -1,16 +1,49 @@
 // Framework and third-party non-ui
-import React from 'react';
+import React, { useContext } from 'react';
 
 // App components
+import TrailCard from 'components/TrailCard';
+import { MapContext } from 'contexts/MapContext';
 
 // JSON & Styles
-import { StyledTrailsList } from './TrailsList-styled';
 
 // Third-party components (buttons, icons, etc.)
+import { VariableSizeList as List } from 'react-window';
+import AutoSizer from 'react-virtualized-auto-sizer';
+
+const getItemSize = (index) => 125;
 
 const TrailsList = () => {
+  const { featureList, setSelection } = useContext(MapContext);
+
+  // TODO: needs polish
+  if (!featureList.length) return 'No features found!';
+
+  const Row = ({ index, style }) => {
```

Comment:
@michaelgaigg these `TrailCards` get re-rendered every time the list is scrolled, plus every time the window is resized... is that expected?